### ISSUE TITLE
[@uplo/service-gcs] Uint8Array not implemented but looks supported ?

### ISSUE THREAD

**Issue by iRyusa** [jpalumickas/uplo#13]
Hi !

I don't know if i'm doing something wrong but in Bun I do : 

```javascript
await uplo.attachments.user(user.id).avatar.attachFile({
            fileName: file.name,
            content: Buffer.from(await file.arrayBuffer()),
            contentType: file.type
          })
```

The problem is: 
`Buffer.from(await file.arrayBuffer()) instanceof Uint8Array` returns  `true`

So it throws because of this:
```javascript
} else if (content instanceof Uint8Array) {
  throw new Error("Uint8Array not implemented");
}
```

If I comment it, the file is uploaded properly tho, is there any reason you can't send a Buffer to GCS ?

**Comment by jpalumickas** [comment]
Hey!

Good question, last time I checked, GCS was not supported Unit8Array, but yea, we can try to impement this or just remove this line.

**Comment by iRyusa** [comment]
Do you want a PR for this ? I'm trying to avoid fork+publish it possible 😅

**Comment by jpalumickas** [comment]
Fixed in v0.20.0 🙂
